Dockerfile.buildtests

```diff
@@ -1,28 +1,29 @@
 # Alpine
 FROM alpine:3.10 AS buildtest-alpine310-static
-RUN apk add --no-cache git build-base autoconf automake libtool linux-headers glib-dev glib-static libcap-static libcap-dev libseccomp-dev
+RUN apk add --no-cache git build-base autoconf automake libtool linux-headers glib-dev glib-static libcap-static libcap-dev libseccomp-dev git meson
 COPY . /src
 WORKDIR /src
+RUN git clone https://gitlab.freedesktop.org/slirp/libslirp.git && cd libslirp && meson setup --default-library=both build && ninja -C build install
 RUN ./autogen.sh && ./configure LDFLAGS="-static" && make && cp -f slirp4netns /
 
 # Ubuntu
 FROM ubuntu:18.04 AS buildtest-ubuntu1804-common
-RUN apt update && apt install -y automake autotools-dev make gcc libglib2.0-dev libcap-dev libseccomp-dev
+RUN apt update && apt install -y automake autotools-dev make gcc libglib2.0-dev libcap-dev libseccomp-dev git meson
 COPY . /src
 WORKDIR /src
+RUN git clone https://gitlab.freedesktop.org/slirp/libslirp.git && cd libslirp && meson setup build && ninja -C build install
 RUN ./autogen.sh
 
 FROM buildtest-ubuntu1804-common AS buildtest-ubuntu1804-dynamic
 RUN ./configure && make && cp -f slirp4netns /
 
-FROM buildtest-ubuntu1804-common AS buildtest-ubuntu1804-static
-RUN ./configure LDFLAGS="-static" && make && cp -f slirp4netns /
```

Comment:
any specific reason to remove this?

Reply:
it's in the commit message:
    Also drop ubuntu-static in docker build (meson there is a bit old and
    doesnt build both dynamic & static at the same time, it shouldn't
    matter as we are targeting shared library in 20.04, and ubuntu should
    have libslirp in backports for 18.04 if necessary)